### PR TITLE
fun little kage hats requested by chat

### PR DESCRIPTION
**Pull request by AlexKJones**
just a few memes kage hats requested by chat

### CODE REVIEW

File: memes.php [hunk 1]
```diff
@@ -534,5 +534,35 @@
         'text' => ':lsm giving himself a medal:',
         'public' => true,
     ],
+    [
+            'code' => ':hatcloud:',
+            'image' => "<img src='./images/memes/hats/hat-cloud.png' />",
```
Comment (levimeahan) [inline memes.php:539]:
```suggestion
            'image' => "<img src='./images/memes/hats/hat-cloud.png' style='max-height:70px;' />",
```

File: memes.php [hunk 2]
```diff
@@ -534,5 +534,35 @@
...
+            'text' => ':basically a neat kite:',
+            'public' => true,
+    ],
+    [
+            'code' => ':hatleaf:',
+            'image' => "<img src='./images/memes/hats/hat-leaf.png' />",
```
Comment (levimeahan) [inline memes.php:545]:
```suggestion
            'image' => "<img src='./images/memes/hats/hat-leaf.png'  style='max-height:70px;' />",
```

File: memes.php [hunk 3]
```diff
@@ -534,5 +534,35 @@
...
+            'text' => ':this hat is on fiaaaahhhh',
+            'public' => true,
+    ],
+    [
+            'code' => ':hatmist:',
+            'image' => "<img src='./images/memes/hats/hat-mist.png' />",
```
Comment (levimeahan) [inline memes.php:551]:
```suggestion
            'image' => "<img src='./images/memes/hats/hat-mist.png' style='max-height:70px;' />",
```

File: memes.php [hunk 4]
```diff
@@ -534,5 +534,35 @@
...
+            'text' => ':there must be a mist opportunity here:',
+            'public' => true,
+    ],
+    [
+            'code' => ':hatsand:',
+            'image' => "<img src='./images/memes/hats/hat-sand.png' />",
```
Comment (levimeahan) [inline memes.php:557]:
```suggestion
            'image' => "<img src='./images/memes/hats/hat-sand.png' style='max-height:70px;' />",
```

File: memes.php [hunk 5]
```diff
@@ -534,5 +534,35 @@
...
+            'text' => ':i hate this hat its so coarse and it gets into everything:',
+            'public' => true,
+    ],
+    [
+            'code' => ':hatstone:',
+            'image' => "<img src='./images/memes/hats/hat-stone.png' />",
```
Comment (levimeahan) [inline memes.php:563]:
```suggestion
            'image' => "<img src='./images/memes/hats/hat-stone.png'  style='max-height:70px;' />",
```

**Review by levimeahan — APPROVED** [review]

**Comment by levimeahan** [comment]
Sweet, thanks for adding these!

**Comment by jayantamadhav** [comment]
Codes don't work on chat

**Pull request merged**